webrtc.html

```diff
@@ -7541,6 +7541,29 @@ <h3>Simulcast functionality</h3>
         therefore need to be able to correctly order the received packets, recognize potential loss events and
         react to them. Correct operation in this scenario is non-trivial and therefore is optional for
         implementations of this specification.</p>
+        <section id="encodingbehavior">
+           <h3>Parameter behavior</h3>
+           <section id="priorityinsimulcast">
+             <h3>priority</h3>
+             <p><code><a>RTCRtpSendParameters</a></code> dictionary member <code>priority</code>
+             sets the priority of an <code><a>RTCRtpSender</a></code>, which influences the
+             bandwidth allocation among <code><a>RTCRtpSender</a></code> objects.</p>
```

Comment:
The "priority" is already per-send parameters, not per encodings, so I'm not sure what this subsection is trying to say clarify that is specific to simulcast.

If the UA is free to decide how to allocate between different streams on the same sender, it should say that.
If the UA has to prioritize all streams with the same priority, it should say that.

Right now I'm not sure what the difference is between two senders with one encoding each and one sender with two encodings. Can you clarify?